frontend/src/metabase-lib/Question.ts

```diff
@@ -757,6 +757,21 @@ class QuestionInner {
       }
     }
 
+    if (
```

Comment:
I've seen this pattern before `(added.length === 0 && removed.length > 0)`. Can we update this to handle the case where both arrays are not empty?

Reply:
Yeah, probably. I just took the inverse of the if statement directly above this `if` statement. I wasn't sure if there was a specific reason why we only did things if 1 of the 2 categories changed, but in general we can probably have logic to say if either of these groups have changed, then update `graph.metrics`.